src/Command/CommandBase.php

```diff
@@ -1257,16 +1261,9 @@ public function setUpdateClient(\GuzzleHttp\Client $client): void {
    */
   public function getUpdateClient(): \GuzzleHttp\Client {
     if (!isset($this->updateClient)) {
-      $stack = HandlerStack::create();
-      $stack->push(new CacheMiddleware(
-        new PrivateCacheStrategy(
-          new DoctrineCacheStorage(
-            new FilesystemCache(sys_get_temp_dir())
-          )
-        )
-      ),
-        'cache');
-      $client = new \GuzzleHttp\Client(['handler' => $stack]);
+      // Cloud API does not send any cache headers, so we won't add any
+      // cache middleware to Guzzle. It wouldn't do anything.
+      $client = new \GuzzleHttp\Client();
```

Comment:
@grasmash this appears to be removing caching from self-update checks. That seems undesirable.

Your code comment is about Cloud API, but I don't see how this method relates to Cloud API?